Sources/TestingMacros/CMakeLists.txt

```diff
@@ -0,0 +1,72 @@
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2024 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See http://swift.org/LICENSE.txt for license information
+# See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+
+cmake_minimum_required(VERSION 3.28)
```

Comment:
Mind lowering this version to match the other one? Thanks.

Reply:
Oh good catch.